app/shared/app-platform/src/commonMain/kotlin/ui/foundation/theme/Color.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2024-2025 OpenAni and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license, which can be found at the following link.
+ *
+ * https://github.com/open-ani/ani/blob/main/LICENSE
+ */
+
+package me.him188.ani.app.ui.foundation.theme
+
+import androidx.compose.ui.graphics.Color
+
+val DefaultSeedColor = Color(0xFF4F378B)
+val AnimekoIconColor = Color(0, 88, 160)
+val BangumiNextIconColor = Color(240, 145, 153)
```

Comment:
This should also be moved to AniThemeDefaults extensions?

We have tootwo places:
- app/shared/ui-foundation/src/commonMain/kotlin/ui/theme/Color.kt
- app/shared/app-platform/src/commonMain/kotlin/ui/foundation/theme/Color.kt